### DISCUSSION TITLE
No system notifications on Android

### DISCUSSION THREAD

**Discussion by Funeoz** [community/community#24084] [Apps, API and Webhooks]
<p>Whenever a new notification is on GitHub, I don’t get any system notifications and this since the first versions of GitHub on Android. The notifications aren’t disabled in the settings.</p>
<p>GitHub version : 1.4.11<br>
Android version : 7.1.1</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Have you enabled push notifications in your app?</p>
<p>We have a guide here:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#enabling-push-notifications-with-github-for-mobile" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#enabling-push-notifications-with-github-for-mobile" target="_blank" rel="noopener">Configuring notifications - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#enabling-push-notifications-with-github-for-mobile" target="_blank" rel="noopener">//docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Please note we only have push notifications in the app for direct mentions at present.</p>

**Comment by Funeoz** [comment]
<p>Oh I thought that it was for all notifications. So then it’s fine.</p>
